src/test/scala/latis/input/TestHapiJsonAdapter.scala

```diff
@@ -0,0 +1,31 @@
+package latis.input
+
+import java.net.URI
+
+import latis.metadata.Metadata
+import latis.model._
+import latis.output.TextWriter
+import org.junit.Test
+import org.scalatest.junit.JUnitSuite
+
+class TestHapiJsonAdapter extends JUnitSuite {
```

Comment:
We're trying to move away from JUnit and adopt `FlatSpec` instead. Take a look at the [tests for `Time`][time] in LaTiS 3.

[time]: https://github.com/latis-data/latis3/blob/27fa46c2cb26e090fb9af0626a52e3e20f8e02f7/core/src/test/scala/latis/time/TimeSpec.scala

Reply:
Cool. Done.